### PR TITLE
Model and View Integration

### PR DESCRIPTION
**Pull request by sravuri-nisum-com**

### CODE REVIEW

File: backboneModel-view/model-view.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+        defaults: {
+            name: 'Guest User',
+            age: 30,
+            occupation: 'worker'
+        }
+    });
+    
+    // A List of People(Collections)
+    App.Collections.People = Backbone.Collection.extend({
+        model: App.Models.Person
```
Comment (vllmstl) [inline backboneModel-view/model-view.js:25]: some kind of collection validations would be good but it's fine for now.

File: backboneModel-view/model-view.js [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    });
+    
+    
+    // View for all people
+    App.Views.People = Backbone.View.extend({
+        tagName: 'ul',
+        
+        initialize: function() {
+            this.collection.on('add', this.addOne, this);//listeners/anouncers for the collection on add..
```
Comment (vllmstl) [inline backboneModel-view/model-view.js:34]: Memory leak, https://coderwall.com/p/fpxt4w/using-backbone-s-new-listento

File: backboneModel-view/model-view.js [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+    });
+    
+    // The View for a Person
+    App.Views.Person = Backbone.View.extend({
+        tagName: 'li',
+    
+        template: template('personTemplate'),	
+        
+        initialize: function(){
+            this.model.on('change', this.render, this);
```
Comment (vllmstl) [inline backboneModel-view/model-view.js:56]: Memory leak, https://coderwall.com/p/fpxt4w/using-backbone-s-new-listento

File: backboneModel-view/model-view.js [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+        // Adding a destroy announcer..
+        },
+        
+        events: {
+         'click .edit' : 'editPerson',
+         'click .delete' : 'DestroyPerson'	//Binding a Destroy for the listing to click event on delete button..
+        },
+        
+        editPerson: function(){
+            var newName = prompt("Please enter the new name", this.model.get('name'));
```
Comment (vllmstl) [inline backboneModel-view/model-view.js:67]: Try creating a new view to get the name :)

**Review by vllmstl — APPROVED** [review]

File: backboneModel-view/model-view.js (hunk 4 above)
Comment (sravuri-nisum-com) [reply backboneModel-view/model-view.js:67]: sure

**Pull request merged**